cvmfs/network/jobinfo.cc

```diff
@@ -0,0 +1,17 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include "jobinfo.h"
+#include "util/string.h"
+
+namespace download {
+
+bool JobInfo::IsFileNotFound() {
+  if (HasPrefix(*url, "file://", true /* ignore_case */))
+    return error_code == kFailHostConnection;
+
+  return http_code == 404;
+}
```

Comment:
I think this is not used anywhere and hence should be removed.

Reply:
Short search says:
It is used here
https://github.com/cvmfs/cvmfs/blob/943eff2c581788030cb3f679d3e7cc64cc3d8a85/cvmfs/publish/repository.cc#L179
and here
https://github.com/cvmfs/cvmfs/blob/943eff2c581788030cb3f679d3e7cc64cc3d8a85/cvmfs/publish/repository.cc#L248

Reply:
Ah, missed these two. Never mind then.